src/ch00-06-constructor.md

```diff
@@ -0,0 +1,15 @@
+# Constructor
+
+Constructors are a special type of function that runs only once when deploying a contract, and can be used to initialize the state of the contract.
```

Comment:
Perhaps you could reword this part with your own understanding of how constructors work in Cairo - we don't need to be super detailed but copy-pasting the existing definition in the Cairo book feels a bit redundant.

Idea:
> A constructor is an optional function that is executed upon contract creation. You can have at most one constructor in a contract, that must be annotated with the `#[constructor]` attribute...